tests/Resources/app/AppKernel.php

```diff
@@ -66,8 +66,9 @@ public function loadRoutes(LoaderInterface $loader): RouteCollection
         $collection->add('/', new Route('/', ['_controller' => 'kernel::indexAction']));
 
         $routes = new RoutingConfigurator($collection, $kernelLoader, $file, $file);
-        $routes->import('@WebProfilerBundle/Resources/config/routing/wdt.xml')->prefix('_wdt');
-        $routes->import('@WebProfilerBundle/Resources/config/routing/profiler.xml')->prefix('_profiler');
+        $extension = self::MAJOR_VERSION >= 7 ? 'php' : 'xml';
```

Comment:
strictly speaking, this is about the version of the WebProfilerBundle which might diverge from the HttpKernel component which provides the symfony version. but this is only for tests, so imho good enough as is.

```suggestion
        // TODO hardcode xml when we support only Symfony 7.3 or higher.
        $extension = self::MAJOR_VERSION >= 7 ? 'php' : 'xml';
```